src/components/orderstable/orderstable.js

```diff
@@ -0,0 +1,132 @@
+import {useState, useEffect} from 'react';
+
+const NUM_ROWS = 3;
+
+export function OrdersTable() {
```

Comment:
You should add a parameter here for `address` then send it to the backend as a data source to populate.  The `address` can be pulled from `orderData.get("walletAddr")` (see page.js for sample codes).

That object is automatically managed/updated using react's `useState`

Reply:
Okay added, but I would like to figure out a way to test.